src/components/ToggleButton.jsx

```diff
@@ -0,0 +1,27 @@
+import { array, string, func } from "prop-types";
+import styles from "./ToggleButton.module.css";
+
+export default function ToggleButton({ options, value, onSelect }) {
+  return (
+    <div className={styles["button-group"]}>
```

Comment:
```suggestion
    <div className={styles.buttonGroup}>
```
I'm not 100% sure, but I think you can use this syntax instead.

Reply:
For some reason it is not working like that.